AppKit/CPWindow/_CPWindowView.j

```diff
@@ -353,7 +353,10 @@ _CPWindowViewResizeSlop = 3;
     if ([theWindow isFullPlatformWindow] ||
         !(_styleMask & CPResizableWindowMask) ||
         (CPWindowResizeStyle !== CPWindowResizeStyleModern))
+    {
+        [[CPCursor arrowCursor] set];
```

Comment:
A sample Xcode project that shows what I'm talking about is attached. Run it and keep the console visible in Xcode so you can see a long of the methods being called. When you move the cursor into the inner view on the left, the cursor stays the same, because it's calling [super cursorUpdate:theEvent], which passes to the next responder, which is its superview. When you move the cursor into the inner view on the right, then into its superview, the cursor does not change because cursorUpdate does not do anything.
[NSTrackingArea.zip](https://github.com/cappuccino/cappuccino/files/53369/NSTrackingArea.zip)